### DISCUSSION TITLE
Changing the background image

### DISCUSSION THREAD

**Discussion by toorajpro** [unfolding-io/nebulix#35] [Q&A]
I am trying to change the background image.
I see that the main image is called `nemulala.jpg` and is located under `src\assets`.
I changed that image to another image of 1024x1024 pixels, and then ran `npm run build` and then `npm run preview`. But when I change the size of the browser I see sometimes the new image I changed the image to is shown and for some sizes, the original image is shown. Do I need to do anything else?

Thanks.

**Comment by preetamslot** [comment]
This is because of the cache. Clear the .cache folder. or use a new file name.

Let me know if this helps
